### ISSUE TITLE
unit testing

### ISSUE THREAD

**Issue by danieleades** [intlify/vue-i18n-composable#2]
what's the correct approach for unit testing using this library?

given this component-

```html
<template>
	<div>
                {{ t("some.translation") }}
	</div>
</template>
```

```typescript
import { defineComponent } from "@vue/composition-api";
import { useI18n } from "vue-i18n-composable";

export default defineComponent({
	setup() {
		const { t } = useI18n();
		return { t };
	},
});
```

i get an error when i shallow-mount this in a unit test

```typescript
import "@/plugins/composition-api";
import { shallowMount } from "@vue/test-utils";
import Component from "@/views/Component.vue";

describe("Component", () => {
	it("can be mounted", () => {
		expect(() => {
			shallowMount(Component);
		}).not.toThrow();
	});
});
```

at the top of a long error output i see `[Vue warn]: Error in data(): "Error: vue-i18n not initialized"`

how should vue-i18n be initialised for this unit test?

**Comment by antfu** [comment]
Which test framework you are using? If you are using Jest you can check out https://jestjs.io/docs/en/setup-teardown

**Comment by axe-me** [comment]
I got similar trouble with my unit tests. 
The createI18n function configures the global Vue, but in the unit test, I need to config localVue.
Is that possible to inject vue instance to the function as a second optional param?
```ts
export function createI18n(options?: VueI18n.I18nOptions, vue: VueConstructor = Vue) {
  vue.use(VueI18n)
  i18nInstance = new VueI18n(options)

  return i18nInstance
}
```
like this.
I can create a PR if you think this is acceptable.

**Comment by antfu** [comment]
@kazupon I think @axe-me's solution is quite reasonable for testing, do you think we can have it? and how could we align this with vue-i18n-next?

**Comment by kazupon** [comment]
I've just published v0.2.0
https://www.npmjs.com/package/vue-i18n-composable
